util/functions.js

```diff
@@ -56,14 +55,16 @@ module.exports = (client) => {
     // the bot is functional, if a user deletes the default settings from the database
     // ensuring the data will re-set those defaults.
     client.settings.ensure("default", defaultSettings);
-    return {
-      ...(client.settings.get("default") || {}),
-      ...(guild && client.settings.get(guild.id) || {})
-    };
+    if(!guild) return client.settings.get("default");
+    const guildConf = client.settings.get(guild.id) || {};
+    // This "..." thing is the "Spread Operator". It's awesome!
+    // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax
+    return ({...client.settings.get("default"), ...guildConf});
   };
 
   // writeSettings overrides, or adds, any configuration item that is different
   // than the defaults. This ensures less storage wasted and to detect overrides.
+  const _ = require("lodash");
```

Comment:
Why did you move the require to down here? By standard it should be on top. It was unnecessary to move this.